jnigen/example/pdfbox_plugin/jnigen_full.yaml

```diff
@@ -22,41 +21,17 @@ preamble: |
   // See the License for the specific language governing permissions and
   // limitations under the License.
 
-## Root for generated C bindings.
```

Comment:
Are these comments not useful?

Reply:
It refers to jnigen.yaml, which has the explanation. This file is only for generating full package bindings instead of 3 classes. (Maybe this can be just a different command using --override or -D, and better to delete the file).